onnx/defs/controlflow/defs.cc

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "onnx/defs/schema.h"
+using namespace onnx;
+
+using SupportType = onnx::OpSchema::SupportType;
+
+std::vector<std::string> alltypes = {"tensor(float)",
+                                     "tensor(int32)",
+                                     "tensor(string)",
+                                     "tensor(bool)",
+                                     "tensor(uint8)",
+                                     "tensor(int8)",
+                                     "tensor(uint16)",
+                                     "tensor(int16)",
+                                     "tensor(int64)",
+                                     "tensor(float16)",
+                                     "tensor(double)"};
+
+OPERATOR_SCHEMA(If)
+    .SetSupportLevel(SupportType::EXPERIMENTAL)
+    .SetDoc("If conditional")
+    .Input(0, "cond", "Condition for the if", "B")
```

Comment:
should we have Output spec too?

and add more details on requirements for then/else branches (e.g. outputs should match)